lib/beaker/dsl/install_utils.rb

```diff
@@ -478,11 +478,18 @@ def do_install hosts, opts = {}
             host['dist'] = "puppet-enterprise-#{version}-#{host['platform']}"
           elsif host['platform'] =~ /windows/
             version = host[:pe_ver] || opts['pe_ver_win']
+            should_install_64bit = !(version_is_less(version, '3.4')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
             #only install 64bit builds if
             # - we are on pe version 3.4+
             # - we do not have install_32 set on host
             # - we do not have install_32 set globally
-            if !(version_is_less(version, '3.4')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
+            if !(version_is_less(version, '4.0'))
```

Comment:
@Iristyle puppet-agent will be versioned starting at 1 soon-ish.  it looks like version comes from line :480 above, which would be the PE version, I'm not sure where `:pe_ver` comes from in this automation, so I'm not sure if this will be an issue, or if we're okay until the shift to "shallow gravy" versioning